src/Components/CreateImageWizard/steps/Repositories/components/UploadRepositoryLabel.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import { Label, Tooltip } from '@patternfly/react-core';
+import { UploadIcon } from '@patternfly/react-icons';
+
+const UploadRepositoryLabel = () => {
+  return (
+    <Tooltip content="Upload repository: Doesn't have an URL, snapshots are taken when content is uploaded to the custom repo.">
```

Comment:
i'm not nearly as familiar with this code as the IB team but this overall looks great to me 😄 

one thing i'm unsure about is the copy here. i'm not sure if the part about not having a URL is important or confusing for users (since there is still the snapshot URL). 

maybe we could have it be similar to what we have in the content sources UI: `Upload repository: Snapshots will only be taken when new content is uploaded.` maybe @regexowl might have some thoughts here too?

Reply:
yeah, copy is not my strong suit so I am unsure about that as well, any suggestions are welcome 😄 _(your suggestion already sounds better)_ 😅

Reply:
awesome, thank you!! this looks great to me 🎉

Reply:
Excellent, thank you! :heart: